crates/ruma-events/src/dummy.rs

```diff
@@ -6,7 +6,9 @@ use std::fmt;
 
 use ruma_macros::EventContent;
 use serde::{
-    de::{self, Deserialize, Deserializer},
+    de::{
+        Deserialize, Deserializer, {self},
```

Comment:
Did you put `self` in curly brackets or was it rustfmt?

Reply:
Must have been rustfmt 😄